cluster/external/provider.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+function _kubectl() {
+    kubectl "$@"
```

Comment:
It a lot of assumptions here:
- your local machine must have `kubectl` binary
- `kubectl` binary must use the correct `KUBECONFIG`

So you need to specify it under README or fetch it from an external cluster(I prefer second option)

Reply:
Sure, this does require some configuration on the part of the user, but it also requires the user to explicitly state the provider, so there's little danger of somebody doing this by accident.  How about I have it fail out with a useful message if there isn't enough info configured?

Reply:
I think that it is just consistent that everything needs is prepared outside of kubevirt, since it is an `external` provider.

Reply:
I added some error messaging.  Hopefully this is good enough

1. Normal behavior:
```
bash-3.2$ KUBEVIRT_PROVIDER=external make cluster-up
./cluster/up.sh
Cluster is up
```
2. Misconfigured cluster (simulated by modifying my ~/.kube/config file)
```
bash-3.2$ KUBEVIRT_PROVIDER=external make cluster-up
./cluster/up.sh
The connection to the server 10.193.228.153:6444 was refused - did you specify the right host or port?

*** Unable to reach external cluster.  Please check configuration ***
*** Type "kubectl config view" for current settings               ***

make: *** [cluster-up] Error 1
```
3. Missing **kubectl** binary
```
bash-3.2$ PATH=/usr/bin KUBEVIRT_PROVIDER=external make cluster-up
./cluster/up.sh
cluster/external/provider.sh: line 4: kubectl: command not found

*** Unable to reach external cluster.  Please check configuration ***
*** Type "kubectl config view" for current settings               ***

make: *** [cluster-up] Error 1
```